src/components/popups/ConfirmPopup.vue

```diff
@@ -0,0 +1,104 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useStoreSettings } from '@/store/settings';
+import InfoIcon from '@/components/icons/Info.vue';
+import CloseIcon from '@/components/icons/XIcon.vue';
+
+const showInfoBubble = ref<boolean>(false);
+const isPosting = ref<boolean>(false);
+const emit = defineEmits(['close', 'post']);
+const settingsStore = useStoreSettings();
+
+function handlePost(): void {
+	if (isPosting.value) {
+		return;
+	}
+	emit(`post`);
+	isPosting.value = true;
+}
+</script>
+
+<template>
+	<div
+		class="popup w-full bg-darkBG dark:bg-gray5 modal-animation fixed top-0 bottom-0 left-0 right-0 z-30 flex h-screen w-full items-center justify-center bg-opacity-50 dark:bg-opacity-50"
+	>
+		<!-- Container -->
+		<section>
+			<div
+				class="relative bg-lightBG dark:bg-darkBGStop card-animation z-10 mr-5 overflow-y-auto rounded-lg p-6 pt-4 shadow-lg"
```

Comment:
`w-full lg:w-600` is missing here

Reply:
What I get when I apply `lg:w-600` . Screen size. `(2560 by 1600)`

<img width="1622" alt="Screenshot 2022-09-13 at 15 58 32" src="https://user-images.githubusercontent.com/60175531/189907322-f587838a-9993-4248-893b-eb60cf5057db.png">

Reply:
Not on the overlay, on the white container

Reply:
`lg:w-600` is just for the white container of the popup, not the entire component

Reply:
Ooooh gotcha